tests/SampleApps/php/greetings/index.php

```diff
@@ -0,0 +1,9 @@
+<?php
```

Comment:
@kichalla  … I added this file for our integration test … had to adjust the require_once path require_once __DIR__ . '/vendor/autoload.php';